lib/ansible/cli/doc.py

```diff
@@ -1124,22 +1111,21 @@ def add_fields(text, fields, limit, opt_indent, return_values=False, base_indent
                         conf['cli'].append(cli)
                 del opt['cli']
 
+            # add custom header for conf
             if conf:
                 text.append(DocCLI._dump_yaml({'set_via': conf}, opt_indent))
 
+            # these we handle at the end of generic option processing
             version_added = opt.pop('version_added', None)
             version_added_collection = opt.pop('version_added_collection', None)
 
+            # general processing for options
             for k in sorted(opt):
                 if k.startswith('_'):
                     continue
-                if isinstance(opt[k], string_types):
-                    text.append('%s%s: %s' % (opt_indent, k,
-                                              textwrap.fill(DocCLI.tty_ify(opt[k]),
-                                                            limit - (len(k) + 2),
-                                                            subsequent_indent=opt_indent)))
-                elif isinstance(opt[k], (Sequence)) and all(isinstance(x, string_types) for x in opt[k]):
-                    text.append(DocCLI.tty_ify('%s%s: %s' % (opt_indent, k, ', '.join(opt[k]))))
+
+                if isinstance(opt[k], Sequence) and not isinstance(opt[k], string_types):
+                    text.append(DocCLI._dump_yaml({k: opt[k]}, opt_indent, flow_style=True))
```

Comment:
For some reason this causes
```
        {aliases: [attr]}
        {choices: [all, 'no', none, safe, urllib2, 'yes']}
        {default: [200]}
```
while non-sequence values for `default` yield
```
        default: 30
```
without the `{...}`.

Reply:
just need to yaml_dump the value and use 'stringification' direclty on key